src/app/server/Server.h

```diff
@@ -66,7 +66,6 @@ enum class PairingWindowAdvertisement
 };
 
 SecureSessionMgr & SessionManager();
```

Comment:
Can we (followup?) remove this `SessionManager()` function and the extern decl in chip-message-send and the decl in `src/app/server/SessionManager.h` and the impl in `Server.cpp`?  No one uses it.

Reply:
Done.